sdk/setup.py

```diff
@@ -1,10 +1,14 @@
 from setuptools import setup, find_packages
+import os
 
 setup(
     name="rapid-sdk",
-    version="0.1.6",
+    # version=os.getenv("TEST_SDK_VERSION", "0.1.6"),
+    version="0.1.6"
```

Comment:
Can we tweak this code a bit to make it a bit cleaner.

Probably want the base version to be a variable above so we only need to change it once and then this inline if statement can read a bit cleaner.

Also probably worth pulling out the test sdk version into a variable above aswell